pkg/provider/kubernetes/ingress/kubernetes.go

```diff
@@ -484,6 +484,11 @@ func (p *Provider) updateIngressStatus(ing *netv1.Ingress, k8sClient Client) err
 			}
 		}
 
+	case corev1.ServiceTypeExternalName:
```

Comment:
Could you please update the documentation to add `ExternalName` to the supported service types list?